test/Project.toml

```diff
@@ -10,6 +10,7 @@ CubedSphere = "7445602f-e544-4518-8976-18f8e8ae6cdb"
 DiffEqBase = "2b5f629d-d688-5b77-993f-72d75c75574e"
 ForwardDiff = "f6369f11-7733-5829-9624-2563aa707210"
 GaussQuadrature = "d54b0c1a-921d-58e0-8e36-89d8069c0969"
+GilbertCurves = "88fa7841-ef32-4516-bb70-c6ec135699d9"
```

Comment:
You're not actually using it directly in the tests, so it doesn't need to be a test dependency